packages/ember-data/lib/system/store.js

```diff
@@ -86,6 +88,20 @@ if (!Backburner.prototype.join) {
 }
 
 
+//Get the materialized model from the internalModel/promise that returns
+//an internal model and return it in a promiseObject. Useful for returning
+//from find methods
+function promiseRecord(internalModel, label) {
```

Comment:
It's a wrapper between the internal world and the outside world so not sure how to make it nicer. But i don't like it